profile/profile.js

```diff
@@ -48,16 +48,13 @@ form.addEventListener('submit', async e => {
     const message = data.get('message');
 
     const profileInfo = await getMyProfile();
-    const profile = await getProfile(id);
     const senderId = profileInfo.id;
```

Comment:
When I try to send a message it's saying cannot read properties of null reading id on this line(line 51). It seems like maybe `profileInfo` isn't showing up?